interactive-calculator/src/App.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class App extends Component {
+  state = {
+    total: "",
+    service:'',
+    partySize: '',
+    totalTip: '',
+    tipPP: ''
```

Comment:
these should all be numeric values. using strings is OK in javascript, but in the long run causes a lot of grief & confusion because of coercion issues.